### PR TITLE
fix: remove obs Epid config

### PR DESCRIPTION
**Pull request by deer-hang**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
remove obs csi bucket epid
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/obs/services/buckets.go [hunk 1]
```diff
@@ -278,8 +277,11 @@ func getObsClient(c *config.CloudCredentials) (*obs.ObsClient, error) {
 	httpClient := c.CloudClient.HTTPClient
 	httpClientConfigure := obs.WithHttpClient(&httpClient)
 
+	userAgent := c.CloudClient.UserAgent.Join()
+	userAgentConfigure := obs.WithUserAgent(userAgent)
```
Comment (chengxiangdong) [inline pkg/obs/services/buckets.go:281]:
```suggestion
	userAgentConfigure := obs.WithUserAgent(config.UserAgent)
```

Comment (deer-hang) [reply pkg/obs/services/buckets.go]: done

**Pull request merged**